views/play.erb

```diff
@@ -0,0 +1,13 @@
+<h1><%= @player.name %>, chose your weapon</h1>
+<form>
+  <h2><input type="radio" name="weapon" value="Rock">
+  <label for="Rock"><%= 'Rock' %></label><br></h2>
+  <h2><input type="radio" id="Paper" name="weapon" value="Paper">
+  <label for="Paper"><%= 'Paper' %></label><br></h2>
+  <h2><input type="radio" id="Scissors" name="weapon" value="Scissors">
+  <label for="Scissors"><%= 'Scissors' %></label></h2>
+</form>
+
+<form action="/result">
```

Comment:
It's common practice for form actions to use the POST method. If a method is not specified it will default to GET